Gemfile

```diff
@@ -26,6 +26,8 @@ gem 'delayed_job_active_record'
 gem 'daemons'
 gem 'foreman'
 gem 'c2', github: '18f/c2-api-client-ruby'
+gem 'jquery-rails-cdn'
```

Comment:
Good point @jessieay . I would only vote local version jquery if the specific version was necessary, which I don't think is the case with MP.